infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/RemoveNodeWorkflow.java

```diff
@@ -0,0 +1,74 @@
+package org.corfudb.infrastructure.orchestrator.workflows;
+
+import com.google.common.collect.ImmutableList;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.orchestrator.Action;
+import org.corfudb.infrastructure.orchestrator.ActionStatus;
+import org.corfudb.infrastructure.orchestrator.IWorkflow;
+import org.corfudb.protocols.wireprotocol.orchestrator.RemoveNodeRequest;
+import org.corfudb.protocols.wireprotocol.orchestrator.Request;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.protocols.wireprotocol.orchestrator.Request'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)